Added JSON-RPC Polygon methods to Polygon Docs (#820)
maticnetwork/matic-docs: docs/develop/api-documentation/gettting-started.md

---
id: getting-started
title: Getting Started with RPC API Methods
---

> JSON-RPC is a stateless, light-weight remote procedure call (RPC) protocol that is commonly used for interacting with the Polygon network layer. As an integral part of the Blockchain Interaction Layer within the [Web3 ecosystem](https://blog.alchemy.com/blog/web3-stack/?a=matic-docs), Polygon JSON-RPC methods allow developers to query for data, write to the blockchain, and build dApps.

Reviewer: Can extend the definition of JSON-RPC to be more clear, as in: 
```
JSON-RPC is a stateless, light-weight remote procedure call (RPC) protocol that is encoded in JSON.
```
I also think if we are adding an intro-level guide, it should point out some additional fundamentals. For example, a user would benefit from:
- general info about RPCs, then lead into JSON-RPC (perhaps on other approaches)
- pointing out if there is a difference between a call and method
- why the Polygon network uses JSON-RPC (e.g. easy to implement)

Better practice would be using internal docs or written pieces to reference the Web3 ecosystem, instead of using the alchemy blog post

> JSON-RPC is a stateless, light-weight remote procedure call (RPC) protocol that is commonly used for interacting with the Polygon network layer. As an integral part of the Blockchain Interaction Layer within the [Web3 ecosystem](https://blog.alchemy.com/blog/web3-stack/?a=matic-docs), Polygon JSON-RPC methods allow developers to query for data, write to the blockchain, and build dApps.

Reviewer: ```suggestion
JSON-RPC is a stateless, light-weight remote procedure call (RPC) protocol that is commonly used for interacting with the Polygon network layer. Polygon JSON-RPC methods allow developers to query for data, write to the blockchain, and build dApps.
```

## API Documentation

> To get started with the API, [visit a complete set of API documentation for standard Polygon JSON-RPC calls](https://docs.alchemy.com/alchemy/apis/polygon-api/?a=matic-docs).

Reviewer: same idea here, should link to internal docs

> To get started with the API, visit the complete set of API documentation for standard [Polygon JSON-RPC calls](https://docs.alchemy.com/alchemy/apis/polygon-api/?a=matic-docs).

Reviewer: ```suggestion
To get started with the API, visit the complete set of API documentation for standard [Polygon JSON-RPC calls](https://edge-docs.polygon.technology/docs/get-started/json-rpc-commands/).
```

## No-Code API Requests

> If you want to get started with API requests with no set-up, fix failing requests, or explore new methods on the Polygon network, try out the [Composer App](https://composer.alchemyapi.io?composer_state=%7B%22chain%22%3A2%2C%22network%22%3A401%2C%22methodName%22%3A%22eth_getBlockByNumber%22%2C%22paramValues%22%3A%5B%22latest%22%2Cfalse%5D%7D).

Reviewer: ```suggestion
If you want to get started with API requests that require no set-up, fix failing requests, or, explore new methods on the Polygon network, try out the [Composer App](https://composer.alchemyapi.io?composer_state=%7B%22chain%22%3A2%2C%22network%22%3A401%2C%22methodName%22%3A%22eth_getBlockByNumber%22%2C%22paramValues%22%3A%5B%22latest%22%2Cfalse%5D%7D).
```

> Composer allows developers to save hours of debugging and building time while also providing a friendly UI for sharing API calls.

Reviewer: ```suggestion
```
